cli/cli.rb

```diff
@@ -217,6 +217,9 @@ def start_services
       fsevents.up
     end
     dns = Dnsmasq.new(machine)
+    if (preferences[:dinghy_domain])
+        dns.dinghy_domain = preferences[:dinghy_domain]
```

Comment:
You can do something like this here:

`dns.dinghy_domain = preference[:dinghy_domain] if preferences[:dinghy_domain]`